packages/lodestar/src/eth1/impl/ethers.ts

```diff
@@ -143,79 +167,71 @@ export class EthersEth1Notifier extends (EventEmitter as { new(): Eth1EventEmitt
    *
    * Returns true if processing was successful
    */
-  public async processBlock(blockNumber: number): Promise<boolean> {
+  public async processBlock(blockNumber: number, blockDepositEvents: IDepositEvent[]): Promise<boolean> {
```

Comment:
Even though we fetched deposit events by range we are still fetching block by block even though we might not need it.

I would propose renaming this method into "processDepositEvents" and break eth1 block voting stuff into "processEth1Votes" (fetching block creating Eth1Data and storing into db). "processEth1Votes" would be only called if there are enough deposits (more than min validators) and spare us from useless block fetching.